xml/troubleshooting.xml

```diff
@@ -1079,64 +1079,25 @@
   </sect2>
 
   <sect2 xml:id="sec-trouble-login-dk">
-   <title>Login Successful but &gnome; Desktop Fails</title>
+   <title>&gnome; Desktop Has Issues</title>
    <para>
-    If this is the case, it is likely that your &gnome; configuration files
-    have become corrupted. Some symptoms may include the keyboard failing to
-    work, the screen geometry becoming distorted, or even the screen coming up
-    as a bare gray field. The important distinction is that if another user
-    logs in, the machine works normally. It is then likely that the problem can
-    be fixed relatively quickly by simply moving the user's &gnome;
-    configuration directory to a new location, which causes &gnome; to
-    initialize a new one. Although the user is forced to reconfigure &gnome;,
-    no data is lost.
+     If you are experiencing issues with the &gnome; desktop (for example,
+     incorrect screen geometry, blank desktop, etc.), try resetting &gnome; settings.
+     Before you do this, back up the existing configuration files using the following command:
    </para>
-   <procedure xml:id="proc-trouble-login-dk">
-    <step>
-     <para>
-      Switch to a text console by pressing <keycombo>
-      <keycap function="control"/> <keycap function="alt"/> <keycap>F1</keycap>
-      </keycombo>.
-     </para>
-    </step>
-    <step>
-     <para>
-      Log in with your user name.
-     </para>
-    </step>
-    <step>
-     <para>
-      Move the user's &gnome; configuration directories to a temporary
-      location:
-     </para>
-<screen>&prompt.user;mv .gconf  .gconf-ORIG-RECOVER
-&prompt.user;mv .gnome2 .gnome2-ORIG-RECOVER</screen>
-    </step>
-    <step>
-     <para>
-      Log out.
-     </para>
-    </step>
-    <step>
-     <para>
-      Log in again, but do not run any applications.
-     </para>
-    </step>
-    <step>
-     <para>
-      Recover your individual application configuration data (including the
-      Evolution e-mail client data) by copying the
-      <filename>~/.gconf-ORIG-RECOVER/apps/</filename> directory back into the
-      new <filename>~/.gconf </filename> directory as follows:
-     </para>
-<screen>&prompt.user;cp -a .gconf-ORIG-RECOVER/apps .gconf/</screen>
-     <para>
-      If this causes the login problems, attempt to recover only the critical
-      application data and reconfigure the remainder of the applications.
-     </para>
-    </step>
-   </procedure>
+<screen>mkdir ./old-gnome-config/ &amp;&amp; mv ./.gnome* ./old-gnome-config/ &amp;&amp; mv .gconf* ./old-gnome-config/ &amp;&amp; mv ./.cache ./old-gnome-config/ &amp;&amp; mv ./dbus ./old-gnome-config/ &amp;&amp; mv ./.dmrc ./old-gnome-config/ &amp;&amp; mv ~/.config/dconf/* ./old-gnome-config/</screen>
+   <para>
+     Run the <command>dconf reset -f /org/gnome/</command>, then log out and try to log in again.
+   </para>
+   <para>
+     If this fixed the desktop issues, restore the desired configuration files.
```

Comment:
Finding out which ones are actually "desired" might be a bit troubling for Mr Joe Random User. Even I don't _really_ know.

Reply:
True, it is troubling. I usually create a fresh new user to exclude all possibly configuration issues in the first place. If that troubleshoots the issue, I add the problematic $HOME directory items to the new $HOME incrementally and find out which one caused the issue. Then it is probably easier to find out the directory of interests. Using this method, I would not be bothered by losing data in the problematic $HOME (I majorly use cp/rsync instead mv now).

However I can barely find a good place to insert this idea nicely in the guide and it may over lengthen the guide. So I am just putting it here for your reference in case it is useful.

Reply:
Maybe this is actually the safest and most efficient way to troubleshoot the problem in the first place? Create a new user, and then "add the problematic $HOME directory items to the new $HOME incrementally and find out which one caused the issue."

Reply:
Even with that approach, I think it might be an issue that dconf is rather hard to separate into different pieces. Also not sure whether all your old config will work again when drop it in piece-meal, there may be dependencies between different dirs. But maybe my criticism are just an example for the perfect being the enemy of the good. :)